admission_control/admission_control_service/src/lib.rs

```diff
@@ -14,12 +14,11 @@
 pub mod admission_control_node;
 /// AC gRPC service.
 pub mod admission_control_service;
+/// Useful Mocks
+pub mod mocks;
 use lazy_static::lazy_static;
 use metrics::OpMetrics;
 
 lazy_static! {
     static ref OP_COUNTERS: OpMetrics = OpMetrics::new_and_registered("admission_control");
 }
-
-#[cfg(test)]
-mod unit_tests;
```

Comment:
we don't need this, as [admission_control_service.rs](https://github.com/libra/libra/pull/962/files#diff-82e6dc122d003df23a96065b3227fca7R41) already includes the unit test file directly.